libs/partners/google-genai/langchain_google_genai/chat_models.py

```diff
@@ -443,7 +443,7 @@ class ChatGoogleGenerativeAI(BaseChatModel):
     top_k: Optional[int] = None
     """Decode using top-k sampling: consider the set of top_k most probable tokens.
        Must be positive."""
-    top_p: Optional[int] = None
+    top_p: Optional[float] = None
```

Comment:
https://medium.com/@mryklief21/guide-to-gemini-pro-for-python-563811b3086e
![image](https://github.com/langchain-ai/langchain/assets/15604894/0a635399-393b-4cc0-a2a4-846817541af1)